src/dlgConnectionProfiles.cpp

```diff
@@ -1528,7 +1542,12 @@ void dlgConnectionProfiles::slot_copyProfile()
         discord_optin_checkBox->setChecked(false);
 
         // restore the password, which won't be copied by the disk copy if stored in the credential manager
-        character_password_entry->setText(oldPassword);
+        // Temporarily block textChanged signal to avoid triggering save on programmatic setText
+        {
```

Comment:
I think the example was misunderstood, { and } are already present in the whole function itself. By putting the signal blocker into another pair of braces just for itself, the signal blocker will only work for these 3 lines and then stop. The solution here is to remove the extra {} that were added.

Reply:
Sure, a QSignalBlocker will only do the blocking as long as it's alive. In this case, because we added a closing brace to line 1549, it'll go out of scope by line 1550 and it'll get deleted, and then it'll stop working. Does that help?

Reply:
Ahh, no. Brainfart on my part, sorry!